pkgs/common-updater/unstable-updater.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, writeScript, coreutils, git, common-updater-scripts }:
+
+# This is an updater for unstable packages that should always use the latest
+# commit.
+{ attrPath
+, url # The git url, usually just src.url
+}:
+
+let
+  updateScript = writeScript "unstable-update-script.sh" ''
+    #! ${stdenv.shell}
+    set -ex
+
+    attr_path="$1"
+    url="$2"
```

Comment:
We might want to consider letting people save typing by computing the URL here from attribute path, at the cost of another evaluation. Hopefully, it will get much cheaper with evaluation cache.

Reply:
Do you mean by running `nix-instantiate` in the script? Well, u-s-v already does it multiple times, so I guess it doesn't hurt to do it one more

Reply:
Should I leave the parameter to let the user override it?

Reply:
I allow that in the old `updateScript`s but only for backwards compatibility, not sure how valuable it really is.

Reply:
Done